### ISSUE TITLE
docs: Show component accessibility status (on leadspace on Usage tab)

### ISSUE THREAD

**Issue by jeanservaas** [carbon-design-system/carbon-website#3505]
### Summary

This is a sibling issue of one that was created earlier for Carbon Platform. https://github.com/carbon-design-system/carbon-platform/issues/803

We want to give users immediate confidence that Carbon components have been vetted for accessibility and are good-to-go out of the box... rather than relying on the user to sift through the Accessibility tab, we need a much more immediate, binary way to display this.

> A crucial aspect of component completeness is knowing to what degree is each asset (e.g. component) accessible. After talking with Marc Johlic, we don't have to go to lengths for each asset to specify which WCAG rules they conform to (e.g. WCAG 2.1 AA.) Instead, accessibility can be reported on a binary basis where an asset is accessible if:
> 
> There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools)
> 
> 1. You can manually keyboard navigate through the asset
> 2. This accessible value could be recorded as: not specified, yes, or no.

We need design resources to:

1. Determine where we'd want to display accessibility status (e.g. leadspace, top of usage tab, LHN)
2. Determine how to display accessibility status (e.g. icon, tag, notification*)

*We probably need to avoid the blue info notifications as we have them everywhere and they tend to get lost.  

### Justification

We've had specific asks from leadership to offer more clarity and emphasis around the accessibility of Carbon out-of-the-box. 

### Specific timeline issues / requests

We need this queued up for the next sprint @sstrubberg

**Comment by laurenmrice** [comment]
Component page with status per component: https://next.carbondesignsystem.com/assets/components
Component status card: https://next.carbondesignsystem.com/libraries/carbon-react/latest/assets/accordion
